VotingApplication/VotingApplication.Web.Api/Scripts/Manage.js

```diff
@@ -16,6 +16,7 @@
 
                 success: function (data) {
                     self.options(data.Options);
+                    self.pollId(data.UUID);
```

Comment:
No, this is the poll Id used to link the user back to the poll they are managing.